ingest/usecase/plugins/orderbook/claimbot/order.go

```diff
@@ -0,0 +1,183 @@
+package claimbot
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/osmosis-labs/osmosis/osmomath"
+	"github.com/osmosis-labs/sqs/domain"
+	"github.com/osmosis-labs/sqs/domain/mvc"
+	orderbookdomain "github.com/osmosis-labs/sqs/domain/orderbook"
+	orderbookgrpcclientdomain "github.com/osmosis-labs/sqs/domain/orderbook/grpcclient"
+	"github.com/osmosis-labs/sqs/log"
+
+	"go.uber.org/zap"
+)
+
+type order struct {
+	Orderbook domain.CanonicalOrderBooksResult
+	Orders    orderbookdomain.Orders
+	Err       error
+}
```

Comment:
This abstraction name is confusing.

This is not an order but rather orderbook with its claimable orders.

I suggest renaming it accordingly as well as the variable identifiers to make the logic more readable.

Reply:
Good catch, updated to `processedOrderbook`.